app/controllers/resources_controller.rb

```diff
@@ -208,7 +220,18 @@ def signed_ids(model_params)
       file_sets(model_params).flat_map do |fileset|
         fileset.dig(:structural, :contains).filter_map do |file|
           # Only include ActiveStorage signed IDs
-          signed_ids[file[:filename]] = file[:externalIdentifier] if decoratable_id?(file[:externalIdentifier])
+          signed_ids[file[:filename]] = file[:externalIdentifier] if signed_id?(file[:externalIdentifier])
+        end
+      end
+    end
+  end
+
+  def globus_ids(model_params)
+    {}.tap do |globus_ids|
+      file_sets(model_params).flat_map do |fileset|
+        fileset.dig(:structural, :contains).filter_map do |file|
+          # Only include ActiveStorage signed IDs
```

Comment:
Is this comment still correct?  (Are globus_ids also ActiveStorage signed_ids?)

Reply:
Good catch, I changed this but didn't commit it.. whoops... pushed now.